sdk/monitor/Azure.Monitory.Query/src/Models/LogsQueryResultRow.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+
+namespace Azure.Monitory.Query.Models
+{
+    public class LogsQueryResultRow
```

Comment:
docs?

Reply:
To come. I want the feature team to take a look an agree on the direction.